modules/oxide/src/less/theme/components/form/textfield.less

```diff
@@ -56,7 +56,8 @@
   .tox-textfield:focus {
     background-color: @textfield-focus-background-color;
     border-color: @textfield-focus-border-color;
-    box-shadow: @textfield-focus-box-shadow;
+    border-radius: 6px;
+    box-shadow: 0 0 0 1px @textfield-focus-border-color;
```

Comment:
Why was this changed from @textfield-focus-box-shadow if we want to change the border here from 2px to 1px then it should alter the variable not a style directly.

Reply:
Reverted